src/modules/locale/LocaleViewStep.cpp

```diff
@@ -281,10 +281,6 @@ LocaleViewStep::setConfigurationMap( const QVariantMap& configurationMap )
     {
         m_localeGenPath = configurationMap.value( "localeGenPath" ).toString();
     }
-    else
-    {
-        m_localeGenPath = QStringLiteral( "/etc/locale.gen" );
-    }
```

Comment:
This only makes sense in the context of changing the documentation that the setting **must** be set; right now the documentation is weirdly formatted, and since the setting is commented out, that suggests that leaving it unset is a sensible thing to do -- that Calamares falls back to something reasonable.

Really the right thing to do is improve the documentation in locale.conf explaining what the possibilityies are (e.g. that SUPPORTED is preferred, and then *localeGenPath* is used, and if that doesn't open, *locale -a* output.)

Reply:
Yes I have an configuartion fix and the fix kkofler poited out for /locale/ modules.

Will push out as soon I'm home

Reply:
Even with these added, I still don't see the point of changing this at all.

I am the original author of the `locale -a` reading code, it works, I don't see why you absolutely want to change it.

Please do not change working code that you don't understand.

Reply:
Why do you think I don't understand how it works ? 

Even you are  author of that code please can you stop 
assuming everyone is stupid but you ?

Also I'm not saying is broken or is wrong! All I say is:

DO not offer configuartion options when they do NOTHING 
and do NOT respect the users settings..

However I drop all these and keep the patches for me ..

Reply:
Sorry, I was somewhat in a bad mood when I wrote that (also because of the recent pull request #849 **from another submitter** (i.e., **not** your fault!) changing documentation without understanding it). I apologize for this unfriendly sentence.

Let me try to get to the technical point now:

> DO not offer configuartion options when they do NOTHING

The option does something: It tells Calamares what file to try to open. If it does not exist, the fallback code is used.

> and do NOT respect the users settings..

The setting is respected: The code tries to open the file, and will use it if it exists. Leaving the option unset is not a documented setting, and it does the (IMHO) most reasonable thing to do, it tries to open the file name as used on the distros actually shipping such a file.